ioc/ticker.db

```diff
@@ -1,15 +0,0 @@
-record(longin, "SR-CS-VEFB-01:TIMER")
```

Comment:
Will we still need these two files (externally) if we want to run with the TFB / VEFB?

Reply:
I don't think so. If I remember correctly even with these two "tickers" running externally it didn't work at all for one and the other was borderline and raised an error eventually. So I think we deemed it easier to just comment out the time checking lines in the feedback code instead.

Reply:
Quite right